sensor_msgs_py/package.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>sensor_msgs_py</name>
+  <version>1.0.0</version>
+  <description>A package for easy creation and reading of PointCloud2 messages in Python.</description>
+  <maintainer email="sebastian.grans@gmail.com">Sebastian Grans</maintainer>
+  <license>Apache License 2.0</license>
+
+  <buildtool_depend>rosidl_default_generators</buildtool_depend>
+
+  <build_depend>builtin_interfaces</build_depend>
+  <build_depend>geometry_msgs</build_depend>
+  <build_depend>std_msgs</build_depend>
```

Comment:
Python packages don't typically have any `build_depend`, so you can remove all of these.